cst/plugins/eu.esdihumboldt.cst.functions.groovy/src/eu/esdihumboldt/cst/functions/groovy/internal/GroovyUtil.java

```diff
@@ -305,6 +306,9 @@ public static Binding createBinding(InstanceBuilder builder, Cell cell, Cell typ
 		binding.setVariable(BINDING_INSTANCE_INDEX,
 				executionContext.getService(InstanceIndexService.class));
 
+		binding.setVariable(BINDING_SERVICE_PROVIDER,
+				executionContext.getService(ServiceProvider.class));
```

Comment:
@emanuelaepure10 I think it must be

```
		binding.setVariable(BINDING_SERVICE_PROVIDER, executionContext);
```

instead as `ExecutionContext` implements the `ServiceProvider` interface. That way, one can do e.g. the following in Groovy scripts to get an instance of `LookupService`:

```
def lookupService = _serviceProvider.getService(LookupService.class)
```